pkgs/top-level/packages-config.nix

```diff
@@ -48,5 +48,15 @@
 
     # This is an alias which we disallow by default; explicitly allow it
     emacs28Packages = emacs28.pkgs;
+
+    # FIXME: if minetestPackages uses callPackage anywhere, this will not work. Why?
```

Comment:
I have no clue why this doesn't work with `callPackage`, I could use some help here